### ISSUE TITLE
[change-to-polkit] Cannot modify 3rd-party repos

### ISSUE THREAD

**Issue by isantop** [pop-os/repoman#28]
On the PolKit branch, 3rd-party repositories cannot be modified.

**Comment by isantop** [comment]
Screenshot of error:

![Screenshot from 2019-11-19 15-02-10](https://user-images.githubusercontent.com/5883565/69190450-ae819700-0add-11ea-9a4b-4d7b6b0636b9.png)

**Comment by isantop** [comment]
As of 86e2c80 we now get:

```
ERROR:dbus.connection:Unable to set arguments (<aptsources.sourceslist.SourceEntry object at 0x7f1940285cd0>, '#deb  http://apt.pop-os.org/staging/rtd3-power-mgmt eoan main') according to signature None: <class 'TypeError'>: Don't know which D-Bus type to use to encode type "SourceEntry"
```

**Comment by isantop** [comment]
Fixed as of ac01978
